apps/session-manager/src/services/__tests__/redis-session-storage.spec.ts

```diff
@@ -505,3 +507,81 @@ describe("RedisSessionStorage#delLollipopDataForUser", () => {
     expect(E.isRight(result)).toBeTruthy();
   });
 });
+
+describe("RedisSessionStorage#deleteUser", () => {
+  const mockedDependencies = {
+    redisClientSelector: mockRedisClientSelector,
+  };
+
+  const TOKENS = [
+    `BPD-${aValidUser.bpd_token}`,
+    `FIMS-${aValidUser.fims_token}`,
+    `MYPORTAL-${aValidUser.myportal_token}`,
+    `SESSIONINFO-${aValidUser.session_token}`,
+    `SESSION-${aValidUser.session_token}`,
+    `WALLET-${aValidUser.wallet_token}`,
+    `ZENDESK-${aValidUser.zendesk_token}`,
+  ];
+
+  test("should succeed deleting all user tokens, if no error occurrs", async () => {
+    mockSrem.mockImplementationOnce((_, __) => Promise.resolve(1));
+    for (let i = 0; i < TOKENS.length; i++)
+      redisMethodImplFromError(mockDel, 1, undefined);
+
+    const response = await pipe(mockedDependencies, deleteUser(aValidUser))();
+
+    expect(response).toEqual(E.right(true));
+
+    expect(mockDel).toHaveBeenCalledTimes(TOKENS.length);
+    expect(mockSrem).toHaveBeenCalledTimes(1);
+
+    for (let i = 0; i < TOKENS.length; i++)
+      expect(mockDel).toHaveBeenNthCalledWith(i + 1, TOKENS[i]);
+    expect(mockSrem).toHaveBeenCalledWith(
+      `USERSESSIONS-${aValidUser.fiscal_code}`,
+      `SESSIONINFO-${aValidUser.session_token}`,
+    );
+  });
+
+  test("should fail when Redis client returns an error on deleting the user tokens", async () => {
+    const errorMessage = "a delete error";
+    const successDeletion = 3;
```

Comment:
the success deletion are 2, otherwise rename the variable as `expectedDeleteCalls`
```suggestion
    const successDeletion = 3;
```